AIO/src/org/aio/activities/banking/ItemReqBanking.java

```diff
@@ -131,6 +131,8 @@ public int execute() throws InterruptedException {
                         setFinished();
                     } else if (!getBank().isOpen()) {
                         getBank().open();
+                    } else if (getBank().getWithdrawMode() != org.osbot.rs07.api.Bank.BankMode.WITHDRAW_ITEM) {
```

Comment:
See my other comment, this logic should be moved to the `getItemReq` function, and we should set the bank node to `WITHDRAW_ITEM` or `WITHDRAW_NOTE` based on `ItemReq.isNoted()` and not always assume that we want to `WITHDRAW_ITEM`